soccer/Referee.cpp

```diff
@@ -410,14 +360,14 @@ GameState Referee::updateGameState(Command command) const {
             restart = GameState::Placement;
             our_restart = !blue_team;
             ball_placement_point = GameState::convertToBallPlacementPoint(
-                ballPlacementx, ballPlacementy);
```

Comment:
Now that all this is multithread with the boost::asio, do we need to worry about mutex's around all this data?

Reply:
yeah.. I think we probably need a mutex whenever`ballPlacementX/Y`, `stage_` or `command_` are used, so like probably in `run()`

Reply:
Actually, on a second thought, there's only one thread that will be accessing this? The only way to do anything is through `run`, and the callbacks (ie. calling `receivePacket`) is done by the thread that calls `_io_service.poll()`

Reply:
@kylestach can you confirm?

Reply:
alright he confirmed its ok

Reply:
No mutex required here, I think. MainWindow should already grab a mutex that processor requires to run while it's offloading data from the GUI thread.